dbt/adapters/contracts/relation.py

```diff
@@ -49,6 +49,7 @@ def __delitem__(self, key): ...
 class RelationConfig(Protocol):
     resource_type: str
     name: str
+    alias: str
```

Comment:
My understanding is that this protocol is representing manifest nodes, which all appear to require `alias` attributes according to the [v12 manifest docs](https://schemas.getdbt.com/dbt/manifest/v12/index.html). If that's the case, I assume it should be fine to expect an `alias` attribute here, but let me know if that's off base and I can adjust it.